channels/qa.m3u

```diff
@@ -19,42 +19,48 @@ https://live-hls-web-ajd.getaj.net/AJD/index.m3u8
 http://ott-cdn.ucom.am/s23/index.m3u8
 #EXTINF:-1 tvg-id="AlJazeeraEnglish.qa" tvg-name="Al Jazeera English" tvg-country="INT" tvg-language="English" tvg-logo="https://i.imgur.com/BB93NQP.png" group-title="News",Al Jazeera English (1080p)
 https://ch.iptvmate.net/b535c29748b3cd2db773e42fdb97672d.m3u8
-#EXTINF:-1 tvg-id="AlJazeeraEnglish.qa" tvg-name="Al Jazeera English" tvg-country="AU" tvg-language="English" tvg-logo="http://i.mjh.nz/.images/tv.46.png" group-title="",Al Jazeera English (1080p) [Geo-blocked]
+#EXTINF:-1 tvg-id="AlJazeeraEnglish.qa" tvg-name="Al Jazeera English" tvg-country="AU" tvg-language="English" tvg-logo="http://i.mjh.nz/.images/tv.46.png" group-title="News",Al Jazeera English (1080p) [Geo-blocked]
 https://i.mjh.nz/au/Sydney/tv.46.m3u8
 #EXTINF:-1 tvg-id="AlJazeeraEnglish.qa" tvg-name="Al Jazeera English" tvg-country="INT" tvg-language="English" tvg-logo="http://www.aljazeera.com/assets/images/aj-logo-lg-124.png" group-title="News",Al Jazeera English
 https://live-hls-web-aje.getaj.net/AJE/01.m3u8
 #EXTINF:-1 tvg-id="AlJazeeraEnglish.qa" tvg-name="Al Jazeera English" tvg-country="INT" tvg-language="English" tvg-logo="https://i.imgur.com/BB93NQP.png" group-title="News",Al Jazeera English (1080p)
 https://live-hls-web-aje.getaj.net/AJE/index.m3u8
-#EXTINF:-1 tvg-id="ALJazeeraMubashe.qa" tvg-name="AL Jazeera Mubashe" tvg-country="QA" tvg-language="" tvg-logo="" group-title="",AL Jazeera Mubashe
+#EXTINF:-1 tvg-id="Al Jazeera Mubasher.qa" tvg-name="Al Jazeera Mubasher" tvg-country="ARAB" tvg-language="Arabic" tvg-logo="https://i.imgur.com/xx6uOhZ.png" group-title="Legislative",Al Jazeera Mubasher
```

Comment:
Nice try, but the tvg-ids do not contain spaces.

Reply:
My bad